workflow-templates/dependabot_approve_and_label.yml

```diff
@@ -92,12 +88,23 @@ jobs:
               pull_number: context.payload.pull_request.number,
               body: "Bumps '${{needs.evaluate_dependabot_pr.outputs.dependency_name}}' from ${{needs.evaluate_dependabot_pr.outputs.version_from}} to ${{needs.evaluate_dependabot_pr.outputs.version_to}}"
             })
+
+  label:
+    runs-on: ubuntu-latest
+    needs: evaluate_dependabot_pr
+    name: Label
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.ENDJIN_BOT_APP_ID }}
+          private_key: ${{ secrets.ENDJIN_BOT_PRIVATE_KEY }}
       - name: 'Label with "autosquash"'
```

Comment:
Again, perhaps a display display making it clearer that this only relates to Dependabot PRs?